service_mesh/v1x/ossm-vs-community.adoc

```diff
@@ -10,16 +10,12 @@ An installation of {SMProductName} differs from upstream Istio community install
 
 The current release of {SMProductName} differs from the current upstream Istio community release in the following ways:
 
-// The following include statements pull in the module files that comprise the assembly.
-
 include::modules/ossm-multitenant.adoc[leveloffset=+1]
 
 include::modules/ossm-vs-istio-1x.adoc[leveloffset=+1]
-[discrete]
-[id="additional-resources_ossm-vs-istio-v1x"]
-[role="_additional-resources"]
-==== Additional resources
 
+[role="_additional-resources"]
+== Additional resources
```

Comment:
Given that this additional resources section is not at the end of the assembly and it seems specific to the preceding section, you could change this to `.Additional resources`.